src/network/meson.build

```diff
@@ -27,6 +27,8 @@ sources = files('''
         netdev/vxlan.h
         netdev/geneve.c
         netdev/geneve.h
+        networkd-address-label.h
+        networkd-address-label.c
         networkd-address-pool.c
```

Comment:
Just a minor nitpick: @keszybz has apparently used .c before .h file consistently in all meson build files, so I would use that sorting for consistencies sake. Also, a small typo in the commit message: s/labbel/label/